utility/lambdas/ecs-asg-sizer/lambda_function.py

```diff
@@ -46,9 +51,9 @@ def slack(text, fields=None):
                 Message=json.dumps(msg),
             )
 
-    # lookup cluster instances, removing any disconnected agents
-    usages = get_usages(ECS_CLUSTER)
-    usages = list(filter(lambda u: u['connected'], usages))
+    # lookup cluster instances and remove disconnected agents
+    all_usages = get_usages(ECS_CLUSTER)
+    usages = list(filter(lambda u: u['disconnectseconds'] < DISCONNECT_THRESHOLD_SECONDS, all_usages))
```

Comment:
Makes sense, filter those usages for a time greater than the threshold. Those are the gone-gone instances.